packages/twenty-front/src/modules/settings/hooks/useExpandedAnimation.ts

```diff
@@ -21,16 +21,19 @@ const advancedSectionAnimationConfig = (
   isExpanded: boolean,
   dimension: AnimationDimension,
   measuredValue?: number,
+  fitContent?: boolean,
 ) => ({
   initial: {
     ...commonStyles(dimension),
   },
   animate: {
     opacity: 1,
     [dimension]: isExpanded
-      ? dimension === 'width'
-        ? '100%'
-        : measuredValue
+      ? fitContent
```

Comment:
Do you really need an additional props here? I feel like `measuredValue` could become `value: number | 'fitContent'`